jmeter-java-dsl-csv-random/jmeter-java-dsl-csv-random.iml

```diff
@@ -0,0 +1,156 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IDE files we avoid uploading to don't tie contributors to any particular IDE, and avoid adding noise. Furthermore, .gitignore should avoid adding *.iml files. Can you please remove this file from the PR?

Reply:
Done